postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -243,7 +248,7 @@
               description: '{{ $labels.instance }} replication slots have not been updated for a significant period, indicating potential issues with replication.',
               summary: 'PostgreSQL replication slots are stale.',
             },
-            expr: 'pg_replication_slots_xmin_age{asserts_env!="", slot_name =~ "^repmgr_slot_[0-9]+"} > 20000',
+            expr: 'pg_replication_slots_xmin_age{slot_name =~ "^repmgr_slot_[0-9]+"} > 20000',
```

Comment:
Another `> 20000`

Reply:
Also removing this alert `PostgresReplicationIsStale`

Since this metric `pg_replication_slots_xmin_age ` does not exist in the exporter.
https://github.com/grafana/postgres_exporter/blob/f59d4af692561afd1dc3d9c749924c8ea7436ae3/cmd/postgres_exporter/postgres_exporter.go#L225